motionplan/motionPlanner.go

```diff
@@ -175,6 +216,8 @@ func PlanWaypoints(ctx context.Context,
 		}
 	}
 
+	logger.Debugf("final plan steps: %v", steps)
```

Comment:
Would it also make sense to log what the motion config is (at least in Debug mode)? We might not otherwise have access to that information without being able to see the software which started us down the Planning path.

Reply:
Yeah that's reasonable, I'll add